pkg/client/client.go

```diff
@@ -405,6 +405,13 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 			return "Disabled"
 		}
 
+		if sr.Masquerading.EnabledProtocols == nil {
+			sr.Masquerading.EnabledProtocols = &models.MasqueradingEnabledProtocols{
+				IPV4: sr.Masquerading.Enabled,
+				IPV6: sr.Masquerading.Enabled,
+			}
+		}
+
 		if !sr.Masquerading.EnabledProtocols.IPV4 && !sr.Masquerading.EnabledProtocols.IPV6 {
 			status = enabled(false)
 		} else {
```

Comment:
Shouldn't the fix be this? :thinking: 
```suggestion
		if sr.Masquerading.EnabledProtocols == nil {
			status = enabled(false)
		} else if !sr.Masquerading.EnabledProtocols.IPV4 && !sr.Masquerading.EnabledProtocols.IPV6 {
			status = enabled(false)
		} else {
```

Reply:
Yeah, it really makes more sense. I will fix that. Thanks @aanm!

Reply:
Actually, the pointer can be nil even when masquerading is enabled. My first code was able to catch this point. Does it make sense?

Reply:
IMHO, the pointer be nil is just a hint that the running agente is not able to inform which protocols masquerading is enabled for. Because it just has a global flag (--masquerading) to enable/disable.

Reply:
@aanm ... what you think about ```status = sr.Masquerading.Enabled```?

Reply:
ah, that makes sense yes!

Reply:
@aanm Done!